chimerapy/engine/utils.py

```diff
@@ -12,7 +12,6 @@
 
 # Third-party
 from tqdm import tqdm
-import ifaddr
```

Comment:
We should remove the ``ifaddr`` dependency. Otherwise, LGTM!

Reply:
This has been removed from the module, but let me remove it from the pyproject.toml